app/helpers/admin_helper.rb

```diff
@@ -0,0 +1,24 @@
+module AdminHelper
+  def show_role(user)
+    if user.enable?
+      if user.user?
+        link_to "Promote to manager", promote_to_manager_admin_admin_path(user), id: "refresh-btn", class: 'btn btn-xs green'
```

Comment:
don't use red and green classes for now, we discussed which classes to use to the other day...i hope you remember.

Reply:
Have changed all button with bootstrap buttons throughout the application.